lib/ap/stdio/ofl.c

```diff
@@ -0,0 +1,16 @@
+#include "stdio_impl.h"
+#include "libc.h"
+
+static FILE *ofl_head;
+//static volatile int ofl_lock[2];
+
+FILE **__ofl_lock()
+{
+	//LOCK(ofl_lock);
+	return &ofl_head;
```

Comment:
APE wasn't thread safe. In fact we haven't any posix thread. This is TODO, because that LOCK iis really a pthread_foo at any part...